modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptNodeClientCodegen.java

```diff
@@ -319,4 +319,37 @@ protected void addAdditionPropertiesToCodeGenModel(CodegenModel codegenModel, Sc
         }
         addImport(codegenModel, codegenModel.additionalPropertiesType);
     }
+
+    @Override
+    public String toDefaultValue(Schema p) {
+        if (ModelUtils.isBooleanSchema(p)) {
+            if (p.getDefault() != null) {
+                return p.getDefault().toString();
+            }
+            return null;
+        } else if (ModelUtils.isDateSchema(p)) {
+            if (p.getDefault() != null) {
+                return p.getDefault().toString();
+            }
+            return null;
+        } else if (ModelUtils.isDateTimeSchema(p)) {
+            if (p.getDefault() != null) {
+                return p.getDefault().toString();
+            }
+            return null;
+        } else if (ModelUtils.isNumberSchema(p) || ModelUtils.isIntegerSchema(p)) {
+            if (p.getDefault() != null) {
+                return p.getDefault().toString();
+            }
+            return null;
+        } else if (ModelUtils.isStringSchema(p)) {
+            if (p.getDefault() != null) {
+                return "'" + (String) p.getDefault() + "'";
```

Comment:
Nit: the string should be escaped

Reply:
That's copy-pasta from `AbstractTypeScriptCodegen` - I'll fix in both places. :)

Reply:
Actually, fixed in `super` and refactored the node codegen to call rather than copy.